wrench/src/main.rs

```diff
@@ -440,10 +468,8 @@ fn main() {
         notifier,
     );
 
-    let mut thing = if let Some(subargs) = args.subcommand_matches("show") {
-        Box::new(YamlFrameReader::new_from_args(subargs)) as Box<WrenchThing>
-    } else if let Some(subargs) = args.subcommand_matches("replay") {
-        Box::new(BinaryFrameReader::new_from_args(subargs)) as Box<WrenchThing>
+    if let Some(subargs) = args.subcommand_matches("show") {
+        render(&mut wrench, &mut window, size, &mut events_loop, subargs);
```

Comment:
does this need to check for the aliases or are they all mapped to "show" by the argument parser?

Reply:
I was a little unsure about this as well, but I confirmed that 'show', 'load', and 'replay' all work.